onnxruntime/core/providers/qnn/qnn_execution_provider.cc

```diff
@@ -1320,4 +1320,12 @@ std::vector<AllocatorPtr> QNNExecutionProvider::CreatePreferredAllocators() {
   return allocators;
 }
 
+OrtDevice QNNExecutionProvider::GetOrtDeviceByMemType(OrtMemType /* em_type */) const {
```

Comment:
Does the QNN EP require a 4K CPU allocator? or any CPU memory is always QNN_HTP_SHARED and that will have 4K alignment by default?

Reply:
Not any more.